app/assets/stylesheets/components/_c-analysis-total.scss

```diff
@@ -0,0 +1,56 @@
+// ANALYSIS TOTAL
+// <div class="c-analysis-total">
+//   <div class="description">Your current estimated emission are:</div>
+//   <div class="total">
+//     <div class="value-box">
+//       <span class="value">144</span>
+//       <span class="info"><svg><use xlink:href="#icon-help"></use></svg></span>
+//     </div>
+//   </div>
+// </div>
+
+.c-analysis-total {
+  padding: rem(40px) 0;
+
+  // Description
+  .description {
+    font-size: rem(32px);
+    color: rgba($font-color-2, 0.8);
+    font-weight: 300;
+    text-align: center;
+  }
+
+  // Total value
+  .total {
+    text-align: center;
+    padding: rem(20px) 0;
+
+    .value-box {
+      position: relative;
+      display: inline-block;
+      margin: 0 auto;
+
+      .value {
+        font-size: rem(120px);
```

Comment:
You also have this one... and all the text items. 
If I missed some of them, feel free to add them there.